File: CastIt.Domain/Dtos/Responses/GetAllPlayListResponseDto.cs
namespace CastIt.Domain.Dtos.Responses
{
    public class GetAllPlayListResponseDto
    {
        public long Id { get; set; }

        public string Name { get; set; }

        public int Position { get; set; }

        public bool Loop { get; set; }

        public bool Shuffle { get; set; }

        public int NumberOfFiles { get; set; }
        public string PlayedTime { get; set; }
        public string TotalDuration { get; set; }
        public string ImageUrl { get; set; }
    }
}

File: CastIt.WatchOS/castIt Watch App/Models/Responses/GetAllPlayListResponseDto.swift
import Foundation

// Mirrors ClientApp IGetAllPlayListResponseDto
struct GetAllPlayListResponseDto: Codable, Identifiable {
    let id: Int
    let name: String
    let position: Int
    let loop: Bool
    let shuffle: Bool
    let numberOfFiles: Int
    let playedTime: String
    let totalDuration: String
    let imageUrl: String
    let lastPlayedDate: String?
}
